service/local-dev/values.yaml.template

```diff
@@ -1,5 +0,0 @@
-imageConfig:
-  imagePullPolicy: IfNotPresent
-global:
-  terraEnv: ENV
-initDB: true
```

Comment:
I'm not sure if `initDB: true` is a good default on personal environments (causes the WSM database to be dropped on server startup). It might actually be more appropriate only on skaffold runs for more db iteration. Or maybe never default to true. @zloery thoughts?

Reply:
Well, this file went away, so does it matter?

In any case, I think this was the right default for "local dev", since the object of local dev to being able to build and launch into your development environment with skaffold.

Reply:
Sorry, more referencing linked PR https://github.com/broadinstitute/terra-helmfile/pull/1384/files#diff-a5a76688e62660b0d4d96da4f210efc1435b881917dde2d9af5191cb78917dd4

Reply:
from personal experience I'd probably default this to false. my workflow generally involved deploying with `initDB` false, and intentionally wiping the DB from time to time instead. I generally set this to true or false explicitly instead of relying on a default, so it also isn't the most impactful to me

Reply:
Oh yeah, I agree. I would not enable that in the helm domain

Reply:
Thanks for the comments all! Would it be best to keep a local `values.yaml` file in this repo for overriding values when deploying with Skaffold? I can make adjustments to the render script to support it.

Reply:
I think it would be useful to keep a local `values.yaml`